project_generation/content/templates/event/config/config.go.tmpl

```diff
@@ -6,20 +6,32 @@ import (
 	"github.com/kelseyhightower/envconfig"
 )
 
+const KafkaTLSProtocolFlag = "TLS"
+
 // TODO: remove hello call config options
```

Comment:
Is it worth moving this comment above KafkaConfig struct?

Reply:
done